core/src/main/java/com/netflix/conductor/service/ExecutionService.java

```diff
@@ -414,11 +414,15 @@ public SearchResult<WorkflowSummary> search(
                                         return new WorkflowSummary(
                                                 executionDAOFacade.getWorkflowById(
                                                         workflowId, false));
-                                    } catch(ApplicationException ae) {
-                                        if(ApplicationException.Code.NOT_FOUND.equals(ae.getCode())) {
-                                            //if workflow not found  in Redis, remove it from ES
-                                            executionDAOFacade.removeWorkflowFromIndexDb(workflowId);
-                                            LOGGER.info("Workflow {} not found both in executionDAO and indexDAO, ignore it", workflowId);
+                                    } catch (ApplicationException ae) {
+                                        if (ApplicationException.Code.NOT_FOUND.equals(
+                                                ae.getCode())) {
+                                            // if workflow not found  in Redis, remove it from ES
+                                            executionDAOFacade.removeWorkflowFromIndexDb(
+                                                    workflowId);
+                                            LOGGER.info(
+                                                    "Workflow {} not found both in executionDAO and indexDAO, ignore it",
```

Comment:
Do we need to say executionDAO and IndexDAO or is it not good to say DB and ES ?